src/game/core/resources/Resources.ts

```diff
@@ -12,27 +17,46 @@ export default class Resources {
       if (this.resourceCache[url]) {
         return this.resourceCache[url];
       } else {
-        const  img = new Image();
-        img.onload = () => {
-          this.resourceCache[url] = img;
-
-          if (this.isReady()) {
-            this.readyCallbacks.forEach(function(func) {
-              func();
-            });
-          }
-        };
-        img.onerror = () => {
+        if (new RegExp(`.(${FILE_EXTENSIONS.IMAGES})$`, 'g').test(url)) {
+          const  img = new Image();
+          img.onload = () => {
```

Comment:
❗️
Так будет читабельнее
```suggestion
          const img = new Image();
          
          img.onload = () => {
```